.github/workflows/ci.yaml

```diff
@@ -29,8 +29,8 @@ jobs:
     strategy:
       matrix:
         os : [ 
-          ubuntu-latest
-          # org-rancher-arm64-containers
+          ubuntu-latest,
+          org--rancher--arm64-containers
           ]
     name : CI
     runs-on : ${{ matrix.os }}
```

Comment:
This becomes:
```
org-${{ github.repository_owner_id }}-{{ arch }}-k8s
```

And then that should start matching the names of the runners we want to use:
`org-9343010-amd64-k8s`